packages/create-sitecore-jss/src/templates/nextjs-multisite/src/lib/middleware/plugins/multisite.ts

```diff
@@ -28,12 +28,24 @@ class MultisitePlugin implements MiddlewarePlugin {
       siteResolver,
       // This function allows resolving site from sc_site cookie, which could be useful in case of Vercel preview URLs. Accepts NextRequest.
       useCookieResolution: () => process.env.VERCEL_ENV === 'preview',
+      // Тhe attributes of the site cookie with recommended values
+      siteCookieAttributes: {
+        secure: true,
+        expires: this.getExpirationDate(10),
+        httpOnly: true,
+        sameSite: 'lax',
+      },
```

Comment:
Wondering why we decided to make this configurable at the application layer? This shouldn't be something a customer need concern themselves with. 

Shouldn't the following attributes satisfy our requirements for this cookie (which we would just set internally)?:
```
secure: true,
httpOnly: true,
sameSite: 'none',
expires: <not set> (i.e. a session cookie)
```

Reply:
Yea, I guess we haven't decided. That question is in the description and I should have raised that up before implementing. I will remove that from the app layer.